### ISSUE TITLE
EXACT syntax needed to work with f2pool 

### ISSUE THREAD

**Issue by fignutz** [trexminer/T-Rex#325]
There are several examples for other types of mining urls provided in the README, but what is the proper syntax to pass to have f2pool see me authenticate? Currently, I'm able to generate a hashrate, but on the f2pool side, they show no workers in the pool. Here is the syntax I'm using (_certain parts redacted_):
```
"pools": [
    {
      "pass": "xxxxxxx",
      "url": "stratum+tcp://eth-us.f2pool.com:6688",
      "user": "[my-f2pool-username].[my-workername]",
      "worker": "[my-workername]"
    }
  ],
```
I'm getting this formatting based off of the f2pool Ethereum guide "**_mine for ASIC systems_**" section, since TREX uses stratum: [f2pool Ethereum guide](https://f2pool.io/mining/guides/how-to-mine-ethereum/)

Should I use a different syntax, and if so , how should my JSON look? On the TREX side "something" is happening that seems productive according to the graph output, but f2pool shows no workers active from my account.

**Comment by pdieppa** [comment]
Have you created a mining account under Account Settings / Mining Accounts?

**Comment by fignutz** [comment]
> 
> 
> Have you created a mining account under Account Settings / Mining Accounts?

Yes. since posting this issue, I've confirmed my account is working via a [lolMiner](https://github.com/Lolliedieb/lolMiner-releases/releases) setup. The lolMiner worker is showing up in my f2pool dashboard.
